recipe/meta.yaml

```
@@ -22,12 +22,13 @@ requirements:
   run:
     - mdtraj
     - numpy
-    - oset
     - scipy
-    - packmol
+    - packmol >=1!18.013
```

@justinGilmer This `packmol` dependency specifier appears to be failing. I think last time I worked on this recipe, we gave up and just removed the specifier (that's why it wasn't in the recipe). I think there was an issue in the particular release `18.013` and someone said we needed to support two different potential channel sources of packmol that used different versioning schemes: one was `1.[stuff]` and the other was `18.[stuff]`. You can see there's disagreement about version schemes for the same package name: https://anaconda.org/search?q=packmol

Maybe you could remove the specifier and keep it as it was, or switch to `>18.013`.
```suggestion
    - packmol
```
```suggestion
    - packmol >18.013
```

---

Ah ok, I forgot about this issue. I think then, we can keep it as is, since that was working before.